ui/pages/add-token/add-token.component.js

```diff
@@ -211,16 +220,20 @@ class AddToken extends Component {
   }
 
   handleCustomDecimalsChange(value) {
-    const customDecimals = value.trim();
-    const validDecimals =
-      customDecimals !== null &&
-      customDecimals !== '' &&
-      customDecimals >= MIN_DECIMAL_VALUE &&
-      customDecimals <= MAX_DECIMAL_VALUE;
-    let customDecimalsError = null;
+    const inValidDecimals = !(
```

Comment:
Could this be `const inValidDecimals = value < MIN_DECIMAL_VALUE || value > MAX_DECIMAL_VALUE`?